Package.resolved

```diff
@@ -14,8 +14,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/apple/swift-log.git",
       "state" : {
-        "revision" : "32e8d724467f8fe623624570367e3d50c5638e46",
-        "version" : "1.5.2"
+        "revision" : "6fe203dc33195667ce1759bf0182975e4653ba1c",
+        "version" : "1.4.4"
```

Comment:
The downgrade is due to supporting earlier platforms?

Reply:
I'm not sure why is it resolving to an earlier version, should be 1.5.2 because of the new CocoaLumberjack 3.8, lemme check.

Nice catch!

Reply:
Done, this is now pointing to the latest version of `swift-log`.